scripts/update-fedora-templates

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+# Utility to update the default Fedora templates used by Qubes
+# when creating other VMs.
+set -e
+set -u
+set -o pipefail
+
+
+# Install the Qubes-maintained Fedora template if absent.
+# We must use conditional logic, since the update command
+# will fail if the package is already installed.
+if ! rpm -qa | grep -q ^qubes-template-fedora-28 ; then
+    sudo qubes-dom0-update qubes-template-fedora-28
+fi
+
+# Check whether a suitable DispVM exists.
+if ! qvm-ls --quiet fedora-28-dvm 2>&1 > /dev/null ; then
+    qvm-create --template fedora-28 --label red fedora-28-dvm
+    qvm-features fedora-28-dvm appmenus-dispvm 1
```

Comment:
I have encountered another error while running  `make all` : an exception is raised 
```
exception template for DispVM (fedora-28-dvm) needs to be an AppVM with template_for_dispvms=True
```

the following command seems to resolve this [0]:
`qvm-prefs fedora-28-dvm template_for_dispvms True`

[0] https://www.qubes-os.org/doc/dispvm-customization/

Reply:
Thanks for reporting, @emkll. Encountered that stipulation in the docs, and was surprised when I didn't need to implement it in the script, during transcription of your requested changes. Let me check my setup and see if I made the mistake of running the action via CLI (which would explain the lack of error)...

Reply:
Was indeed able to reproduce viewing the error; but the `make all` command was still exiting zero. See #51 for tracking those silent failures.